homeassistant/components/nest/camera_sdm.py

```diff
@@ -188,7 +190,10 @@ async def async_camera_image(self) -> bytes | None:
         stream_url = await self.stream_source()
         if not stream_url:
             return None
-        return await async_get_image(self.hass, stream_url, output_format=IMAGE_JPEG)
+        image_bytes: bytes | None = await async_get_image(
```

Comment:
This should be typed upstream in `homeassistant.components.ffmpeg`

Reply:
Thanks, fixed.